### PR TITLE
Request to add two new commits to obiwan_app

### PR DESCRIPTION
**Pull request by pablonap**
* Add
   * new line to  README
   * a file named main.java

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1 +1,2 @@
 # Description of obiwan-app
+# Features
```
Comment (obiwannap) [inline README]: Hi. Sorry mate but for now I don't want to add features to README file.

**Review by obiwannap — CHANGES_REQUESTED** [review]
Pls guys don't add any line to README file.

File: README (hunk 1 above)
Comment (pablonap) [reply README]: It's ok. I'm not gonna add any line to README. :+1:

**Review by obiwannap — APPROVED** [review]
Cool. That's good.

**Pull request merged**